src/theory/uf/theory_uf_rewriter.cpp

```diff
@@ -175,6 +200,10 @@ Node TheoryUfRewriter::rewriteViaRule(ProofRewriteRule id, const Node& n)
       {
         return Node::null();
       }
+      // Note that we do not check for variable shadowing in the lambda here.
+      // This rule will only be used to capture valid instances of beta
```

Comment:
"capture" and "beta reduction" in the same sentence immediately lead one to think of variable capture. Please use a different word. :)